claim-check-pattern/ReadME.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: pattern 
+title: Claim Check Pattern
+folder: Claim-Check-Pattern 
+permalink: /patterns/claim-check-pattern/ 
+description: Cloud Architecture
+categories: Cloud
+tags:
+ - Cloud distributed
+ - Microservices
+---
+
+## Name
+[Claim Check Pattern](https://docs.microsoft.com/en-us/azure/architecture/patterns/claim-check)
+
+## Also known as
+[Reference-Based Messaging](https://www.enterpriseintegrationpatterns.com/patterns/messaging/StoreInLibrary.html)
+
+## Intent
+- Reduce load of data transfer through internet
+- Data security
+
+## Explaination
```

Comment:
```suggestion
## Explanation
```

Reply:
The text seems to contain a lot of grammar and spelling issues. Can you please run the text through a text checking service such as grammarly.com and fix the errors?

Reply:
@iluwatar used Grammarly now.

Reply:
@iluwatar done